templates/blog/article.html

```diff
@@ -0,0 +1,133 @@
+{% extends "base.html" %}
+
+{% block meta_title %}{{ article.title.rendered|safe }}{% endblock %}
+
+{% block meta_description %}{{ article.meta_description }}{% endblock %}
+
+{% block body_class %}
+  is-paper blog-article
+{% endblock body_class %}
+
+{# djlint: off #}
+  <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@id": "https://canonical.design/#article",
+      "@type": "Article",
+      "name": "{{ article.title.renderered|safe }}",
+      "headline": "{{ article.meta_description }}",
+      "author": {
+        "@type": "Person",
+        "name": "{{ article.author.name }}"
+      },
+      "datePublished": "{{ article.date_gmt }}",
+      {% if article.image and article.image.source_url %}
+      "image": "{{ article.image.source_url }}",
+      {% endif %}
+      "url": "{{ request.url }}",
+      "publisher": {
+        "@type": "Organization",
+        "name": "Ubuntu"
+      }
+    }
+</script>
+{# djlint: on #}
+
+{% block content %}
+
+  {% set breadcrumbs = [{"name": "Blog", "href": "/blog"}, {"name": "Article"}] %}
+  {% include '/shared/_breadcrumbs.html' %}
+
+  <article class="p-section">
+    <header class="row">
+      <div class="aside col-3 col-medium-2">
+        <div class="p-media-object">
+          {% if article.author %}
+            {% if article.author and article.author.id == 217 %}
+              <img src="https://assets.ubuntu.com/v1/f16c40d0-Favicon+-+CoF.svg"
+                   class="p-media-object__image"
+                   alt=""
+                   style="align-self: center" />
+            {% else %}
+              <img src="{{ article.author.avatar_urls['96'] }}"
+                   class="p-media-object__image is-round"
+                   alt=""
+                   style="align-self: center" />
+            {% endif %}
+          {% endif %}
+
+          <div class="p-media-object__details">
+            <p class="u-no-margin--bottom">
+              {% if article.author %}
+                <span class="p-heading--5">
+                  <a href="/blog/author/{{ article.author.slug }}"
+                     title="More about {{ article.author.name }}">{{ article.author.name }}</a>
+                </span>
+                <br />
+              {% endif %}
+              <span>on {{ article.date }}</span>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="p-block col-9 col-medium-4">
+        <div class="p-block">
+          <h1>{{ article.title.rendered|safe }}</h1>
+        </div>
+        {% if tags %}
+          <div class="p-block">
+            {% for tag in tags %}
+              <a class="p-chip--information" href="/blog/tag/{{ tag.slug }}">
+                <span class="p-chip__value">{{ tag.name }}</span>
+              </a>
+            {% endfor %}
+          </div>
```

Comment:
```suggestion
          <div class="p-block">
            {% set rendered_tags = [] %}
            {% for tag in tags %}
              {% if tag.name not in set_tags %}
                <a class="p-chip--information" href="/blog/tag/{{ tag.slug }}">
                  <span class="p-chip__value">{{ tag.name }}</span>
                </a>
              {% endif %}
              {% set rendered_tags = rendered_tags.append(tag.name) %}
            {% endfor %}
          </div>
```
You can do something this to handle the duplicate tags